cohorts.md

```diff
@@ -0,0 +1,265 @@
+---
+layout: page
+title: All Open Life Science Cohorts till Date!
+image: /images/syllabus.jpg
+photos:
+  name: Niklas Morberg
+  license: CC BY-NC 2.0
+  url: https://flic.kr/p/5BXB6s
+---
+
+# Cohorts
+
+<div class='cohort-grid'>
+  <div class="flip-card">
+  <div class="flip-card-inner">
+    <div class="flip-card-front">
+    <h1>OLS - 1</h1>
+    </div>
+    <div class="flip-card-back">
+   <a href='ols-1'><h2>Click here to find out more about the first cohort!</h2></a>
```

Comment:
Which information does it show there?

Reply:
It just takes you to the cohorts page

Reply:
Do you want me to include things to the card like the year, timeline (i mean from 20xx-20xx) or any other thing?

Reply:
Yes it would be good I think